### DISCUSSION TITLE
Перенос информации

### DISCUSSION THREAD

**Discussion by TheParaziT** [PavelBlend/blender-xray#777] [Q&A]
Можно ли перенести информацию из Wiki аддона на мой личный проект по сбору информации по моддингу?
https://github.com/Igigog/anomaly-modding-book

**Comment by PavelBlend** [comment]
@TheParaziT можно. Только я её ещё не дописал и хотел сделать изменений и добавить новых страниц. Ты её на английский будешь переводить?

**Reply by TheParaziT** [reply]
@PavelBlend Да. Потом может быть русскую локализацию сделаю
Тогда буду ждать полной версии
